cpp/dolfinx/io/vtk_utils.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2020-2022 Garth N. Wells and and Jørgen S. Dokken
+//
+// This file is part of DOLFINx (https://www.fenicsproject.org)
+//
+// SPDX-License-Identifier:    LGPL-3.0-or-later
+
+#pragma once
+
+#include <cstdint>
+#include <dolfinx/mesh/cell_types.h>
+#include <tuple>
+#include <vector>
+#include <xtensor/xtensor.hpp>
+
+namespace dolfinx::fem
+{
+class FunctionSpace;
+}
+
+namespace dolfinx::mesh
+{
+enum class CellType;
+class Mesh;
+} // namespace dolfinx::mesh
+
+namespace dolfinx::io
+{
+/// Given a FunctionSpace, create a topology and geometry based on the
+/// dof coordinates.
+/// @note Only supports (discontinuous) Lagrange functions
+/// @param[in] V The function space
+std::tuple<xt::xtensor<double, 2>, std::vector<std::int64_t>,
+           std::vector<std::uint8_t>, xt::xtensor<std::int32_t, 2>>
```

Comment:
Maybe add a description of the output. Why are there 4 arrays in the tuple?

I guess it should be:
Returns the mesh geometry `(block_size*(num_dofs_local+num_ghosts, 3)`, the global geometry index, ghost_level marker and the topology (mesh connectivity) using local indices

Reply:
Return object docs added.